bin/deploy-systemtests.sh

```diff
@@ -3,6 +3,7 @@
 
 export PROJECT_ROOT=$(dirname $(dirname $(realpath ${0})))
 
+DOCKER_COMPOSE_CMD="$(command -v docker-compose > /dev/null && echo docker-compose || echo docker compose)"
```

Comment:
Det er ret fjollet. Alle nyere docker versioner er forsynet med en "compose" action. Men well. Så kan vi da også køre den på devel10.

Reply:
Ved det godt - havde noget hejs på 12 og var nødt til at rykke over på 10, så satte den ind så man slap for at rette alle de steder hvor der stod docker compose.
Antager at i "gør noget et eller andet sted i v3".